styles/styles.css

```diff
@@ -487,9 +487,36 @@ pulse.off {
 }
 
 #mathAnnotation {
-    width: 280px; 
-    height: 118px; 
-    margin-top: 1px; 
-    margin-bottom: -6px; 
+    width: 280px;
+    height: 118px;
+    margin-top: 1px;
+    margin-bottom: -6px;
     border-left: 1px solid #CCC;
 }
+
+#mathPalette {
+    flex-grow: 200;
+}
+
+#mathButtons {
+    width: 330px;
+}
+
+.btn-scratch {
+    background-color: white;
+    background-image: url(../images/scratch.png);
+}
+
+.btn-scratch:active {
+    background-image: url(../images/scratch.png)!important;
```

Comment:
Wouldn't it work without overriding, despite it's :active?

Reply:
ok I will delete it